profiles/Leustyin-moths.yaml

```diff
@@ -0,0 +1,22 @@
+---
+
+# Hunting settings
+hunting_info:
+- :zone:  fuligin_moths
+  args:
+  - d0
+  - all
+  :duration: 60
+  before:
+  - go2 XXXXX
+  - weararmor
+  - release mof
+  # - buff main
+  - buff aeg
+  - buff aeg
+  after:
+  - release cyclic
+  stop_on:
+  - Staves
+  kill_on:
```

Comment:
❓ I'm not a fan of the name `kill_on` - I'm not sure it's clear enough what it does. Especially since it can coexist with `stop_on`, I can easily see users being unclear on how they differ. Have you thought of any other possible names for this?

Reply:
stop_when_low: maybe?